src/helpers/useUndo.ts

```diff
@@ -98,17 +110,22 @@ export const useUndo = ({
   nodes,
   edges,
   disabled,
+  initialHistory,
   maxHistory = 20,
   onUndoRedo
 }: UndoProps): UndoResult => {
   const [canUndo, setCanUndo] = useState<boolean>(false);
   const [canRedo, setCanRedo] = useState<boolean>(false);
 
-  const manager = useRef<Undoo>(
-    new Undoo({
-      maxLength: maxHistory
-    })
-  );
+  const undoo = new Undoo({
+    maxLength: maxHistory
+  });
+
+  const manager = useRef<Undoo>(undoo);
+
+  if (Array.isArray(initialHistory)) {
+    undoo.import(initialHistory);
```

Comment:
This need to be in a `useEffect`

Reply:
Using a deps of [] so it only runs once, I guess?

Reply:
Yes